includes/template-library/sources/local.php

```diff
@@ -891,9 +827,11 @@ public function import_template( $name, $path ) {
 				$items[] = $import_result;
 			}
 
-			rmdir( $temp_path );
+			// Delete the temporary extraction directory, since it's now not necessary.
+			rmdir( $extracted_files['temp_extraction_directory'] );
```

Comment:
Try to find a way to handle it by the `Uploads_Manager`

Reply:
same as above.